cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -219,35 +230,6 @@ fn lower_insn_to_regs<'a>(ctx: Ctx<'a>, iri: IRInst) {
             panic!("ALU+imm and ALU+carry ops should not appear here!");
         }
 
-        Opcode::X86Udivmodx
```

Comment:
I feel like it's a shame to lose this, because it means losing the ability to easily differentiate legitimate failures due to non-implementation of a target-independent CLIR insn, vs bugs resulting in machine-specific CLIRs being handed to us.

Reply:
Oh, these `X86*` opcodes went away in the latest `master`, so this is just a rebase-related change. I think the rule is still (as has always been) never have a fallthrough in the big-opcode-match, and handle additions or deletions as they come, indicated by compile errors -- so if any machine-specific ops are added back in the future, we'll figure out what to do then.